index.html

```diff
@@ -3,10 +3,144 @@
+          <p class="home-location-info">@StudentsToWorkd Collage-EEUU</p>
+        </div>
+      </div>
+    </section>
+
+    <section class="activities" id="activities">
+      <div class="activities-subtitle">
+        <h2> Main Program </h2>
+        <hr>
+      </div>
+      <div class="containerBoxs">
+        <div class="boxs">
+          <img src="/images/nuevo-mensaje.png" class="imagesActivity" alt="" srcset="">
+          <h4>Forum</h4>
+          <p>Enjoy taking some time Take the time to share your thoughts and opinions with experts for each topic.</p>
+        </div>
+        <div class="boxs">
+          <img src="/images/taller.png" class="imagesActivity" alt="" srcset="">
+          <h4>Advisory</h4>
+          <p>Get free advice and recommendations from the most experts in the field, negotiate according to your interests.</p>
+        </div>
+        <div class="boxs">
+          <img src="/images/comida.png" class="imagesActivity" alt="" srcset="">
+          <h4>Aperitif</h4>
+          <p>Enjoy a delicious appetizer while you enjoy observing the greatest works of great artists of the genre.</p>
+        </div>
+        <div class="boxs">
+          <img src="/images/tendencia.png" class="imagesActivity" alt="" srcset="">
+          <h4>Trends</h4>
+          <p>Find out about the latest trends, update your knowledge, get ideas, find best and most detailed information.</p>
+        </div>
+        <div class="boxs">
+          <img src="/images/exhibicion.png" class="imagesActivity" alt="" srcset="">
+          <h4>Exhibition</h4>
+          <p>You can enjoy a wide gallery of exhibitions, where you can find a very varied range of works made in different cultures.</p>
+        </div>  
+      </div>
+      <div class="containerButton">
+        <button type="button" class="Button" id="Button">Join Urban Art Festival</button>
+      </div>
+    </section>
+
+    <section class="DinamicsSection">
+      <div class="grid-section" id="artistsContainer">
+ 
+      </div>
+    </section>
+    
+    <section class="parthnerSection">
+      <h2>Partner</h2>
+      <div>
+        <ul class="parthner-conten">
+          <li class="parthnerLi"><img src="/images/parthner/Adobe-Logo-1.png" alt="" srcset=""></li>
+          <li class="parthnerLi"><img src="/images/parthner/RedBull2.png" alt="" srcset=""></li>
+          <li class="parthnerLi"><img src="/images/parthner/amnesty-international.png" alt="" srcset=""></li>
+          <li class="parthnerLi"><img src="/images/parthner/google.png" alt="" srcset=""></li>
+          <li class="parthnerLi"><img src="/images/parthner/patagonia.png" alt="" srcset=""></li>
+        </ul>
+      </div>
+    </section>
+      <div class="footer">
+        <img src="/images/LogoFer.png" alt="" srcset="">
+        <div class="licencedFooter">
+          <p>2023 Fernando Jose Montes Linares. Some Rights Reserved.</p>
+          <p>"ご訪問いただき、ありがとうございます。皆様にとって素晴らしい日々が訪れますように。"</p>
+        </div>
```

Comment:
- [ ] It looks like there might be a little hiccup with the design on mobile. The partner logos seem a bit small, and the footer logo is a tad larger than the template

   ### Your design:
   ![image](https://github.com/Fernando426/CapstoneN01/assets/107952724/7e52dc76-2fc0-4223-b031-1ff4fd197177)

   ### Behance design:
   ![image](https://github.com/Fernando426/CapstoneN01/assets/107952724/26a20468-4246-4ed1-8861-d393e2e34f0a)